scripts/skills/actives/rf_gain_ground_skill.nut

```diff
@@ -0,0 +1,154 @@
+this.rf_gain_ground_skill <- ::inherit("scripts/skills/skill", {
+	m = {
+		ValidTile = null
+	},
+	function create()
+	{
+		this.m.ID = "actives.rf_gain_ground";
+		this.m.Name = "Gain Ground";
+		this.m.Description = ::Reforged.Mod.Tooltips.parseString("Keep going! After killing an enemy you may move into their tile ignoring [Zone of Control|Concept.ZoneOfControl]. Can only be used immediately after a successful kill.");
+		this.m.Icon = "skills/rf_gain_ground_skill.png";
+		this.m.IconDisabled = "skills/rf_gain_ground_skill_sw.png";
+		this.m.Overlay = "rf_gain_ground_skill";
```

Comment:
The overlay of active skills is spawned automatically when the skill is used. I think you wanted to talk about spawning the icon of the Vanquisher perk when landing on a valid tile. I agree that's a good idea and I have added that in a new commit now.

Reply:
I mean, that's also worth considering.

But I meant to spawn an icon when the character kills another characters (and subsequently triggers Vanquisher).
See here in Berserk
![image](https://github.com/Battle-Modders/mod-reforged/assets/2252464/f12994d4-ec97-4052-ad0e-33ec52001f7d)

Reply:
Vanquisher's effect is only triggered when you move into the tile. So I've added it so that the icon is only spawned when you actually move into the tile, not when making a kill. When making a kill you get the Gain Ground skill activated (previously it is black&white) so that is enough of a representation imo.